scrapers_next/ma/committees.py

```diff
@@ -0,0 +1,89 @@
+from spatula import URL, HtmlListPage, HtmlPage, CSS, XPath, selectors
+from openstates.models import ScrapeCommittee
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        com.add_source(
+            self.source.url,
+            note="Committee Details Page",
+        )
+        try:
+            member_blocks = XPath('//*[contains(@class, "committeeMemberList")]').match(
+                self.root
+            )[0]
+        except selectors.SelectorError:
+            return com
```

Comment:
Rewrite this line as `raise SkipItem(f"No membership data found for: {com.name}")`